marshmallow/schema.py

```diff
@@ -486,7 +490,7 @@ def dumps(self, obj, many=None, update_fields=True, *args, **kwargs):
         serialized = self.dump(obj, many=many, update_fields=update_fields)
         return self.opts.render_module.dumps(serialized, *args, **kwargs)
 
-    def load(self, data, many=None, partial=None):
+    def load(self, data, many=None, partial=None, unknown=None):
```

Comment:
Is it necessary/useful to add `unknown` as a parameter to `load`, or will the constructor parameter suffice?

Reply:
I've done it just like `partial` (and `many`), which I think are doing it the right way. I'd say that the more flex you can get, the better it is.